modules/@apostrophecms/module/index.js

```diff
@@ -272,7 +272,7 @@ module.exports = {
       // be used very sparingly, and pretty much never in npm modules. The
       // only exceptions in apostrophe core are `apos.area` and `apos.singleton`.
       //
-      // The helper must be added first with `addHelpers` or `addHelper`.
+      // The helper must be added first with `addHelpers`.
```

Comment:
The helper really should be added by putting it in the helpers section, although addHelpers still exists for implementation purposes, so how about documenting that instead.

Reply:
There's no method by that name.